client/components/errorLabel/errorLabel.css

```diff
@@ -1,4 +1,4 @@
-.button-error {
```

Comment:
You should import typography.css to use font-family by default.
The label now displayed in Times New Roman despite the correct font is specified.
In typography.css defined class error-message. Use it